PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetShapesMC.cxx

```diff
@@ -963,7 +987,7 @@ void AliAnalysisTaskEmcalJetShapesMC::NTValues(AliEmcalJet *jet, Int_t jetContNb
   }
 }
 //_________________________________________________________________________________________________
-Double_t AliAnalysisTaskEmcalJetShapesMC::fjNSubJettiness(AliEmcalJet *Jet, Int_t JetContNb, Int_t N, Int_t Algorithm, Double_t Beta, Int_t Option){
+Double_t AliAnalysisTaskEmcalJetShapesMC::fjNSubJettiness(AliEmcalJet *Jet, Int_t JetContNb, Int_t N, Int_t Algorithm, Double_t Beta, Int_t Option, Double_t Beta_SD, Double_t ZCut, Int_t SoftDropOn){
```

Comment:
I don't recall the approval history but I recommend renaming of this function in order to avoid confusion (mixing naming convention of member function and local data member). There is a good reason for naming conventions.

Reply:
fj here stands for fastjet not the f for local data members. I used this because this function is specifically the fastjet Nsubjettiness function and not the original one which i wrote.

Reply:
Yes, then please use a capital F as it is the standard for function names. It is just one letter to change.